packages/orbit-components/src/Skeleton/Skeleton.stories.tsx

```diff
@@ -18,129 +15,113 @@ enum PRESETS {
   Text = "Text",
 }
 
-export const Default = () => {
-  return <Skeleton height="150px" width="500px" />;
+const meta: Meta<typeof Skeleton> = {
+  title: "Skeleton",
+  component: Skeleton,
 };
 
-Default.story = {
+export default meta;
+type Story = StoryObj<typeof Skeleton>;
+
+export const Default: Story = {
+  render: () => <Skeleton height="150px" width="500px" />,
+
   parameters: {
     info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      disable: true,
+    },
   },
 };
 
-export const Playground = ({
-  animate,
-  height,
-  maxHeight,
-  rowBorderRadius,
-  rowHeight,
-  rowOffset,
-  rows,
-  spaceAfter,
-  viewBox,
-  width,
-  title,
-  color,
-}) => {
-  return (
-    <Skeleton
-      animate={animate}
-      color={color}
-      height={height}
-      maxHeight={maxHeight}
-      title={title}
-      rowBorderRadius={rowBorderRadius}
-      rowHeight={rowHeight}
-      rowOffset={rowOffset}
-      rows={rows}
-      spaceAfter={spaceAfter}
-      viewBox={viewBox}
-      width={width}
-    />
-  );
-};
-
-Playground.story = {
+export const Playground: Story = {
```

Comment:
I would move `Playground` and `RTL` as the last stories as it's in the previous batches we did before.

Reply:
I moved those two stories down to keep consistency. But I am not really sure about reusing the args and argTypes, it's not saving anything, it's just three values 😅

Reply:
I understand your reason, I was pointing to a consistency, as we have the same approach in other stories (if it is not forgotten :D ), where we also "save" just 2-3 args. :D

Reply:
- https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/src/Collapse/Collapse.stories.tsx
- https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/src/CarrierLogo/CarrierLogo.stories.tsx

reusing Playground stories (yes, it's 4 args, not 3 :D ):
- https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/src/Inline/Inline.stories.tsx

Reply:
Maybe I'm missing something, but is there any reason why `preset` is not included in Playground story?

Reply:
Solved, my bad. :D